.gitignore

```diff
@@ -7,6 +7,7 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+/.idea/runConfigurations.xml
```

Comment:
Why did you add this? I think we want to be able to use VCS shared run configurations.